src/style.css

```diff
@@ -0,0 +1,46 @@
+body {
+  margin: 0;
+}
+
+.starsContainer {
+  display: flex;
+  justify-content: center;
+}
+
+.starsField {
+  display: flex;
+  flex-direction: column;
+  width: 122px;
+  height: 96px;
+}
+
+.stars {
+  height: 16px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+}
+
+.stars__star {
+  width: 13px;
```

Comment:
Please use width/height like in mockup 
![image](https://user-images.githubusercontent.com/54936044/222902310-6b5bf4f6-2e57-42e5-a5ca-d149eaaa107a.png)